scripts/flytekit_sagemaker_runner.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import
```

Comment:
🗡️ 
Also, can you add a comment to this script explaining when/where it's run?  Is this something users run from their laptops?